code/modules/crafting/recipes.dm

```diff
@@ -262,6 +262,19 @@
 	time = 100
 	category = CAT_WEAPON
 
+/datum/crafting_recipe/lmusket
+	name = "Laser Musket"
+	result = /obj/item/weapon/gun/energy/l_musket
+	reqs = list(/obj/item/weaponcrafting/reciever = 1,
+				/obj/item/pipe = 1,
+				/obj/item/stack/cable_coil = 1,
+				/obj/item/stack/sheet/plasteel = 1,
+				/obj/item/weapon/shard = 1)
```

Comment:
Where are the electrical components? Could it not require a microlaser (autolathe) or something?